src/header.rs

```diff
@@ -1482,7 +1482,7 @@ fn copy_path_into(mut slot: &mut [u8], path: &Path, is_link_name: bool) -> io::R
     for component in path.components() {
         let bytes = path2bytes(Path::new(component.as_os_str()))?;
         match (component, is_link_name) {
-            (Component::Prefix(..), false) | (Component::RootDir, false) => {
+            (Component::Prefix(..) | Component::RootDir, false) => {
```

Comment:
this changes the minimum rust version, without really helping that much

Reply:
That's fair. Is there a particular MSRV that you're targeting?

Reply:
not really atm, I guess we can go 1.51 for now which is a great baseline, and I think will allow this change in

Reply:
how about i create a separate pull request to test the MSRV in CI, and then we can circle back to this?